docs/tutorials/multimodal/object_detection/inference/detection_inference_coco.md

```diff
@@ -0,0 +1,166 @@
+```
+Currently, we convert the results to a pandas `DataFrame` and save into a `.txt` file.
+The `.txt` file has two columns, `image` and `bboxes`, where
+- in `image`, each row contains the image path
+- in `bboxes`, each row is a list of dictionaries, each one representing a bounding box: 
+  - `{"class": <predicted_class_name>, "bbox": [x1, y1, x2, y2], "score": <confidence_score>}`
+
+## Reading results
+The returned value `pred` is a `list` and has the following dimensions:
+```
+[num_images, num_total_classes, num_detections_per_class, 5]
+```
+
+where 
+- `num_images` is the number of images used to run inference on. 
+- `num_total_classes` is the total number of classes. In this example, `num_total_classes = 20` for VOC dataset
+- `num_detections_per_class` is the number of detections under each class. Note that this number can vary across different classes.
+- The last dimension contains the bounding box information, which follows `[x1, y1, x2, y2, score]` format. `x1, y1` are the top left corner of the bounding box, and `x2, y2` are the bottom right corner. `score` is the confidence score of the prediction
+
+example code to examine bounding box information:
+
+```python
+detection_classes = predictor.get_predictor_classes()
+idx2classname = {idx: classname for (idx, classname) in enumerate(detection_classes)}
+for i, image_pred in enumerate(pred):
+    print("result for image {}".format(i))
+    for j, per_cls_bboxes in enumerate(image_pred):
+        classname = idx2classname[j]
+        for bbox in per_cls_bboxes:
+            ## bbox = [x1, y1, x2, y2, conf_score]
+            print("bbox: {}, class: {}, score: {}".format(bbox[:4], classname, bbox[4]))
+```
+If you prefer to get the results in `pd.DataFrame` format, run the following:
+```python
+pred_df = predictor.predict(test_path, as_pandas=True)
+```
+
+Similar to the `.txt` file, the `pred_df` also has two columns, `image` and `bboxes`, where
+- in `image`, each row contains the image path
+- in `bboxes`, each row is a list of dictionaries, each one representing a bounding box: 
+  - `{"class": <predicted_class_name>, "bbox": [x1, y1, x2, y2], "score": <confidence_score>}`
+
+## Visualizing Results
+To run visualizations, you'll need to setup `cv2` by running 
+```python
+pip install cv2
+```
+
+To visualize the detection bounding boxes, run the following:
+```python
+from autogluon.multimodal.utils import from_coco_or_voc, visualize_detection
+import matplotlib.pyplot as plt
+
+conf_threshold = 0.4  # Specify a confidence threshold to filter out unwanted boxes
+visualization_result_dir = "~/data/coco17/visualizations"  # Specify a directory to save visualized images.
+df = from_coco_or_voc(test_path)[:10][["image"]]
+
+pred = predictor.predict(df)
+
+visualized = visualize_detection(
+    pred=pred,
+    data=df,
+    detection_classes=predictor.get_predictor_classes(),
+    conf_threshold=conf_threshold,
+    visualization_result_dir=visualization_result_dir,
```

Comment:
`visualization_result_dir` is where the machine-labeled images will go?

Reply:
yea this is the directory to save the machine-labeled images